### PR TITLE
Edit partner post

### PR DESCRIPTION
**Pull request by jwebs1**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,21 @@
+0 info it worked if it ends with ok
```
Comment (srveit) [inline npm-debug.log]: Remove this file

File: Partners/partners-requesting-customer-trial.md [hunk 2]
```diff
@@ -1,58 +1,61 @@
...
   "author": "",
   "attachments": [],
   "contentIsHTML": false
 }}}
 
-Overview
-CenturyLink’s Cloud delivers some of the highest performing Infrastructure in the industry (Source: Cloud Harmony Report). 
-Our infrastructure delivers databases with higher IOPS, Websites with faster response times, storage with greater availability, as well as many more benefits that your customer can take advantage of.
-**Offering customer’s trial often lead to wins:**
+## Overview
```
Comment (srveit) [inline Partners/partners-requesting-customer-trial.md]: Change to `###`

File: Partners/partners-requesting-customer-trial.md [hunk 3]
```diff
@@ -1,58 +1,61 @@
...
-* Customer Name 
-* Customer Alias (Portal Alias) 
+* Customer Name
+* Customer Alias (Portal Alias)
 * Trial Start Date
 
-You will get a confirmation email from CenturyLink Cloud Trial Team confirming the Trial. 
+You will get a confirmation email from CenturyLink Cloud Trial Team confirming the trial.
+
+## Emails from CenturyLink and Partner Actions
```
Comment (srveit) [inline Partners/partners-requesting-customer-trial.md]: Change to `###`

Comment (brandysmith) [inline Partners/partners-requesting-customer-trial.md]: We do not use H3s

Comment (srveit) [inline Partners/partners-requesting-customer-trial.md]: @jim-phillips please comment on this

**Pull request merged**